pkg/operator/mgr/mgr.go

```diff
@@ -89,6 +104,31 @@ func (c *Cluster) Start() error {
 	return nil
 }
 
+func (c *Cluster) makeService(name string) *v1.Service {
+	service := &v1.Service{}
+	service.Name = name
+	service.Namespace = c.Namespace
+
+	service.Spec = v1.ServiceSpec{
+		Selector: c.getLabels(),
+		Type:     v1.ServiceTypeClusterIP,
+		Ports: []v1.ServicePort{
+			{
+				Name:     "mgr",
```

Comment:
who do you imagine will be a consumer of the service port for `mgr` itself?  I imagine that the service port for the metrics will be consumed by external scrapers (prometheus), but i'm not sure who would need a service port for `mgr`.

Reply:
Right, I remove the `mgr` port.